ReadMe.md

```diff
@@ -58,7 +58,10 @@ To edit | 編集（する） | へんしゅう |
 Automatic | 自動 | じどう |
 Screen | 画面 | がめん |
 Video | 動画 | どうが |
+Picture | 写真 |　しゃしん
```

Comment:
if we're talking about an image in technical or design terms, its better to use the work 画像
sorry this was also on my list of words to add to the original document that i never got around to.
feel free to update it here @yayannabelle ^_^

fun fact, this can also change in the context of background which would be more correct to use the work 背景